### ISSUE TITLE
JAWS with Chrome does not announce aria-label on table cells with aria-hidden or empty content

### ISSUE THREAD

**Issue by straydogstrut** [FreedomScientific/VFO-standards-support#405]
### Summary

Where an `aria-label` is present on a table cell that has either empty content or content hidden from assistive technology using `aria-hidden=”true”`, the `aria label` is not announced by JAWS.

### Expected result

* The `aria-label` for the cell is announced once.
* The cell content if present or lack of cell content (e.g. "blank") is not announced* 

\* *Though previous implementations seem to disagree with this - see Additional Information below.*

### Actual result

* When the cell's content is hidden from AT using `aria-hidden` or the cell has no content, the cell is announced as "blank".
* When the cell content is not hidden/empty, only the cell content is announced. The `aria-label` is not announced.

### Example

See the [codepen](https://codepen.io/straydogstrut/pen/QWyyvJQ) with the following code:

```
<table>
  <tr>
   <th>Date</th>
    <th>Event</th>
    <th>Venue</th>
    <th>Organiser</th>
    <th>Admission</th>
  </tr>
  <tr>
    <td aria-label="My content is hidden from AT using aria-hidden"><span aria-hidden="true">12 February</span></td>
    <td aria-label="I have no content"></td>
    <td aria-label="I have content and an aria-label">Main Hall</td>
    <td></td>
    <td>Free</td>
  </tr>
</table>
```
- Row 2, Cell 1 – `aria-label` with `aria-hidden` cell content
- Row 2, Cell 2 – `aria-label` with empty cell content
- Row 2, Cell 3 – `aria-label` with cell content
- Row 2, Cell 4 – regular empty cell
- Row 2, Cell 5 – regular cell with content

#### Related Issues/Documentation

- Not considered an Accessible Name Calculation issue ([Hiding Content has no effect on Accessible Name Calculation](https://www.w3.org/TR/using-aria/#hiding-content-has-no-effect-on-accessible-name-or-description-calculation)) as it is the `aria-label` that is not been announced, not the cell content
- `aria-label` is described as supported on `table` elements and these cells have an implicit role. See [Practical Support](https://www.w3.org/TR/using-aria/#label-support)
- [JAWS with Edge reads aria-label multiple times for a table with role="img"](https://github.com/FreedomScientific/VFO-standards-support/issues/174) is similar to behaviour experienced in previous versions of JAWS with Chrome.

#### Other JAWS Versions combined with Chrome

* With JAWS 2018 / 2019 and Chrome, when the cell's content is hidden from AT using aria-hidden or the cell has no content, *the aria-label is announced twice*.
* With JAWS 18 / 2018 / 2019 and Chrome, when the cell content is not hidden/empty, *both the aria-label and cell text are announced once*.

#### JAWS versions combined with IE11

* With JAWS 2018 / 2019 / 2020 and IE11, when the cell content is hidden from AT using aria-hidden or the cell has no content, *the aria-label is announced once as expected*.
* With JAWS 2018 / 2019 / 2020 and IE11, when the cell content is not hidden/empty, *both the aria-label and cell text are announced once*.
* With JAWS 18 and IE11, *the aria-label is not announced in any circumstance (expected behaviour I believe)*.

#### JAWS version and build number

JAWS 2020 2004.66 ILM
JAWS 2020 2006.12 ILM

*Also for reference:*
JAWS 2019 2019.1907 42 ILM
JAWS 2018 2018.1811 30 ILM
JAWS 18 18.0.5038
 
#### Operating System and version

Windows 10

#### Browser and version:

Chrome 83.0.4103.116
